test/postgresql.test.js

```diff
@@ -230,6 +230,121 @@ describe('postgresql connector', function() {
     });
   });
 
+  context('pattern matching operators', function() {
+    before(function deleteTestFixtures(done) {
+      Post.destroyAll(done);
+    });
+    before(function createTextFixtures(done) {
+      Post.create([{
+        title: 't1',
+        content: 'T1_TestCase',
+      }, {
+        title: 't2',
+        content: 'T2_TheOtherCase',
+      }], done);
+    });
+    after(function deleteTestFixtures(done) {
+      Post.destroyAll(done);
+    });
+
+    it('should support case sensitive queries using like',
```

Comment:
Yes, I see the other tests are different (we need to eventually update all of them to this style over time